### PR TITLE
Gtk update dylan

### PR DESCRIPTION
**Pull request by JelleBootsma**
Look at the wiki checklist for any new methods or functionality ([https://github.com/Group8Sep/scroom/wiki/Code-review-list](https://github.com/Group8Sep/scroom/wiki/Code-review-list)

### CODE REVIEW

File: gui/src/gtkruler.c [hunk 1]
```diff
@@ -0,0 +1,913 @@
+/* GTK - The GIMP Toolkit
```
Comment (JelleBootsma) [inline gui/src/gtkruler.c:1]: File is not used and should be removed

File: gui/src/gtkruler.h [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU Library General Public
```
Comment (laurens-kolenbrander) [inline gui/src/gtkruler.h:14]: This file is also not used right?

File: out.txt [hunk 3]
```diff
@@ -0,0 +1,7274 @@
+[  4%] Built target threadpool
+[  5%] Built target boosttesthelper
+[  7%] Built target spmeasure
+[ 11%] Built target util
+[ 15%] Built target sanity_tests
+[ 17%] Built target boosttesthelper_tests
+[ 21%] Built target spcolormap
+[ 29%] Built target threadpool_tests
```
Comment (laurens-kolenbrander) [inline out.txt]: This file also should not be committed I believe

**Review by laurens-kolenbrander — COMMENTED** [review]
There are some files that should be removed before merging

**Pull request closed without merging**